posts/inside-rust/2022-02-22-compiler-team-ambitions-2022.md

```diff
@@ -0,0 +1,424 @@
+
+The compiler team leadership plans to put out a post in June summarizing the progress so far on the items listed here, and another post in November with a retrospective on how the year went.
+
+#### I did not see any mention of monadic burritos (or other non-Rust language feature); why is that not part of your plan?
+
+The scope of this doc is largely restricted to Compiler Team issues. Language design work is done by the Language Team. You can reach out to them about their initiatives for this year and beyond.
+
+#### What do I do if I'm interested in learning more about a specific item on this list?
+
+Each item in this list has one or more owners listed with it. The Rust Compiler team largely communicates via the [Zulip] chat platform.
+
+So: set up a Zulip account, sign into Zulip, and join the [#**new members>compiler 2022**][on zulip] topic. Tell the group which item you're interested in, and also mention the owners listed with that topic so that they know to join you in that conversation channel. We will help you get started from there.
+
+[Rustc Dev Guide]: https://rustc-dev-guide.rust-lang.org/
+[Zulip]: https://rust-lang.zulipchat.com/
+[on zulip]: https://rust-lang.zulipchat.com/#narrow/stream/122652-new-members/topic/compiler.202022
+
+#### What do I do if I'm interested in compiler development but have no experience in compilers?
+
+This is not a problem! Many members of our community learned about compilers by working on rustc, and we encourage others to do so as well. You can start by reading the [Rustc Dev Guide] and by joining us on [Zulip]. You may also benefit from watching the RustConf 2021 presentation on [Contributing to the Compiler] by [estebank].
+
+[Contributing to the Compiler]: https://www.youtube.com/watch?v=vCODCbUSA_w
+
+In addition, there are areas in this project where people without compiler expertise can have impact. For example, as mentioned in the [Performance Dashboard](#Performance-Dashboard) section, some of our internal tools could use some web front-end work.
+
+### How can I contact an item's owners or sponsor their work on Rust?
+
+This table lists the item owners mentioned above, their [Zulip] username and if they are accepting sponsorships to help them work on Rust:
+
+Owner | Zulip Username | Accepting sponsorships?
+-|-|-
+[Aaron Hill] | `@Aaron Hill` | No
+[antoyo] | `@antoyo` | Yes: [GitHub Sponsors](https://github.com/sponsors/antoyo)
+[apiraino] | `@apiraino` | No
+[bjorn3] | `@bjorn3` | Yes: [Librapay](https://liberapay.com/bjorn3)
+[cjgillot] | `@cjgillot` | No
+[davidtwco] | `@davidtwco` | No: works on Rust at Huawei R&D UK
+[estebank] | `@Esteban Küber` | No: works on Rust at Amazon Web Services
+[jackh726] | `@Jack Huey` | No
+[jswrenn] | `@Jack Wrenn` | No: works on Rust at Amazon Web Services
+[lcnr] | `@lcnr` | Yes: https://lcnr.de/funding/
```

Comment:
```suggestion
[lcnr] | `@lcnr` | Yes: [https://lcnr.de/funding/](https://lcnr.de/funding/)
```

(without this, the url doen't render as a hyper link, at least in the blog post rendered preview )